app/controllers/mechanics_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MechanicsController < ApplicationController
+
+  def show
+  @mechanic = Mechanic.find(params[:id])
+  end
+
+  def add_ride
+    mechanic = Mechanic.find(params[:id])
+    ride = Ride.find(params[:ride_id])
+    mechanic.rides << ride
+    redirect_to "/mechanics/#{mechanic.id}"
+  end
```

Comment:
Yup, this logic looks good, just want it to be in a restful action, like an `update`